server/controller/src/main/resources/application.yaml

```diff
@@ -12,6 +12,8 @@ spring:
       static-locations: file:/opt/starwhale.java/static/, classpath:/static/
 sw:
   version: ${SW_VERSION_CONTROLLER:0.1.0:8c82767b60686f3e2bfea9dafe8c8cce5dd34f52}
+  docker:
+    registry: ${SW_DOCKER_REGISTRY_URL:docker-registry.starwhale.cn}
```

Comment:
use empty value as the default?  same as pypi default value.

Reply:
done